scripts/setup-torcharrow.sh

```diff
@@ -0,0 +1,85 @@
+#!/bin/bash
+# Copyright (c) Facebook, Inc. and its affiliates.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -efx -o pipefail
+# Some of the packages must be build with the same compiler flags
+# so that some low level types are the same size. Also, disable warnings.
+export CFLAGS="-mavx2 -mfma -mavx -mf16c -masm=intel -mlzcnt -w -std=c++17"  # Used by LZO.
+export CXXFLAGS=$CFLAGS  # Used by boost.
+
+yum -y install ccache
+yum -y install ninja-build
+yum -y install git
+yum -y install openssl-devel
+yum -y install double-conversion-devel
+yum -y install glog-devel
+yum -y install bzip2-devel
+yum -y install gflags-devel
+yum -y install gtest-devel
+yum -y install libevent-devel
+yum -y install lz4-devel
+yum -y install libzstd-devel
+yum -y install re2-devel
+yum -y install snappy-devel
```

Comment:
I remember @mbasmanova mentioned some of these deps are not required when building `VELOX_MINIMAL` (e.g. lz4/snappy/zstd are only used in Hive Connector). But we can leave this as future work.

Reply:
Yes, I left this purposely out for KR3.